docs/source/reference/core/configuration.rst

```diff
@@ -67,6 +67,14 @@ Note that the default values are set in the global config.
        - If it is ``'never'``, Chainer will never use cuDNN anywhere.
 
    The default value is ``'auto'``.
+``chainer.config.cudnn_use_tensor_core``
+   Flag to configure whether or not to enable Tensor Core operatons in cuDNN.
+
+       - If it is ``True``, Chainer uses cuDNN's Tensor Core operations.
```

Comment:
Could you use the values consistent with other options (e.g. `use_cudnn`)? I think it can be automatically mapped like `True -> 'always'`, `False -> 'never'`, and `None -> 'auto'`.

Reply:
Thanks for your comment. I like consistency, so I think your suggestion is better.
I just updated the branch in that way, also the option name was changed to `use_cudnn_tensor_core` considering consistency.